controllers/ingressnodefirewallconfig_controller.go

```diff
@@ -143,6 +152,14 @@ func (r *IngressNodeFirewallConfigReconciler) syncIngressNodeFwConfigResources(c
 		}
 	}
 
+	if config.Spec.EBPFProgramManagerMode != nil {
+		data.Data["EBPFProgramManagerMode"] = "0"
```

Comment:
Given that we have the constant `ebpfProgramMangerEnvVar` defined:

```sh
pkg/ebpf/ingress_node_firewall_loader.go
39:     ebpfProgramMangerEnvVar       = "EBPF_MANAGEMENT_MODE"
```

wouldn't it be better to use this constant instead of the string literal `"EBPFProgramManagerMode"`? This would reduce duplication and improve consistency by using a single source for the constant string throughout.

Reply:
Given that we have:

```yaml
 - name: EBPF_MANAGEMENT_MODE
              value: '{{.EBPFProgramManagerMode}}'
```

then I drop my suggestion as we would still need this "string literal" in the YAML.